docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ---
 sidebar_position: 2
 ---
 
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# Generating SessionSigs
+# Generating Session Signatures
+
+[Session signatures](./intro.md) can be obtained using the [getSessionSigs](https://v5.api-docs.getlit.dev/classes/lit_node_client_src.LitNodeClientNodeJs.html#getSessionSigs) method provided by the Lit SDK. This method will generate a _session keypair_ for you locally, and will use the `authNeededCallback` method you specify to sign the session keypair, authorizing it to use the specified [Lit Resources and Abilities](./resources-and-abilities.md) on the Lit network.
+
+Below we will walk through an example of generating session signatures, the full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/blob/wyatt/get-session-sigs-via-auth-sig/get-session-sigs-via-auth-sig/nodejs).
+
+## Prerequisites
+
+Before continuing with this guide, you should have an understanding of:
+
+- [Session Signatures](../../authentication/session-sigs/intro.md)
+- [Lit Capacity Credits](../../../sdk/capacity-credits.md)
+- [Lit Resources and Abilities](./resources-and-abilities.md)
+
+This guide also has a dependency on the following packages:
+
+- `"@lit-protocol/auth-helpers": "^6.0.0-beta.3"`
+- `"@lit-protocol/constants": "^6.0.0-beta.3"`
+- `"@lit-protocol/lit-node-client": "^6.0.0-beta.3"`
```

Comment:
This is now `beta.4` perhaps we use a script to update these package versions so we can run it once and update all docs pages to the latest package build.